src/components/projects/projectId/playground/EvaluationGroups.tsx

```diff
@@ -0,0 +1,132 @@
+        if (!projectId) return;
+        if (isFetchingEvalGroups.current) return;
+        isFetchingEvalGroups.current = true
+        getEvaluationGroups(projectId, (res) => {
+            setEvaluationGroups(res);
+            isFetchingEvalGroups.current = false
+        });
+    }, [projectId]);
+
+    useLayoutEffect(() => {
+        if (!selectedEvaluationGroups || !evaluationGroups) return;
+        const isIndeterminate = selectedEvaluationGroups.size > 0 && selectedEvaluationGroups.size < evaluationGroups.length;
+        setChecked(selectedEvaluationGroups.size > 0 && selectedEvaluationGroups.size === evaluationGroups.length);
+        setIndeterminate(isIndeterminate);
+
+        if (checkbox.current !== null) {
+            checkbox.current.indeterminate = isIndeterminate;
+        }
+    }, [selectedEvaluationGroups, evaluationGroups]);
+
+    useEffect(() => {
+        if (!evaluationGroups) return;
+        setPreparedValues(prepareTableBodyEvaluationGroups(evaluationGroups, selectedEvaluationGroups, setSelectedEvaluationGroups, usersDict, viewSetsModal));
+    }, [evaluationGroups, selectedEvaluationGroups, setSelectedEvaluationGroups]);
+
+    function viewSetsModal(groupId: string) {
+        let setsArr = [];
+        getEvaluationSetsByGroupId(projectId, groupId, (res) => {
+            setsArr = res;
+            dispatch(setModalStates(ModalEnum.VIEW_EVALUATION_GROUP, { open: true, sets: setsArr }));
+        });
+    }
+
+    useEffect(() => {
+        setPreparedHeaders(preparedHeaders.map((header) => {
+            if (header.id === "checkboxes") {
+                return { ...header, hasCheckboxes: true, checked: checked, onChange: toggleAll };
+            }
+            return header;
+        }))
+    }, [checked, evaluationGroups, selectedEvaluationGroups])
+
+    const refetchEvaluationGroups = useCallback(() => {
+        getEvaluationGroups(projectId, (res) => {
+            setEvaluationGroups(res);
+            setSelectedEvaluationGroups(new Set<string>());
+        });
+    }, [projectId]);
+
+    function toggleAll() {
+        if (checked || indeterminate) setSelectedEvaluationGroups(new Set<string>());
+        else setSelectedEvaluationGroups(new Set<string>(evaluationGroups.map(x => x.id)));
+        setChecked(!checked && !indeterminate)
+        setIndeterminate(false)
+    }
+
+    return <>
+        {projectId != null && <div className="p-4 bg-gray-100 h-full flex-1 flex flex-col overflow-y-auto">
+            <div className="text-lg leading-6 text-gray-900 font-medium w-full flex items-center">
+                <div>
+                    <label>Evaluation groups</label>
+                    <div className="mt-1 mb-3">
+                        <div className="text-sm leading-5 font-normal text-gray-500 inline-block">You can create your evaluation groups and use them for creating evaluation runs.</div>
+                    </div>
+                </div>
+                <button
```

Comment:
why not kernbutton?

Reply:
@JWittmeyer for refinery we don't use KernButton since the KernButton was added later in the submodules. 
I test it but it looks differently from the buttons that we have in refinery -> some padding and font size are diffent.
I suggest that we to the replace of the buttons in refinery to Kern Button in its own PR, if not I can of course change it with the KernButton

Reply:
later it's own PR works :)